src/transformers/modeling_t5.py

```diff
@@ -953,6 +958,12 @@ def forward(
 
         hidden_states = encoder_outputs[0]
 
+        # If the model is only provided with either input_ids or inputs_embeds,
```

Comment:
This actually does not make much sense - if no `decoder_input_ids` are provided they should not just be set to the `encoder_input_ids` IMO. @mfuntowicz - can we revert this change? or will it break onnx support?